config/services.yaml

```diff
@@ -28,3 +28,9 @@ services:
       - services
 
   PhpMx\Router:
+
+  PhpMx\Services\GetRandomMessageByType:
+    arguments:
+      - '%plus_plus.increased_messages%'

```

Comment:
it is the autowire function available on the project? i think could be a good way to avoid to explicit definition of the every dependency

Reply:
Do you know if the atowire also work for arrays?

Reply:
This comment was removed since the service receives a repository insted of an array, the autowire does the magic ;)